ci/increment-version.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+VERSION=$(git describe --abbrev=0 --tags)
+
+MAJOR=$(echo $VERSION | cut -d. -f1 | sed 's/v//') 
+MINOR=$(echo $VERSION | cut -d. -f2)
+PATCH=$(echo $VERSION | cut -d. -f3)
+
+if [ "$1" == "major" ]; then
+  MAJOR=$((MAJOR+1))
+  MINOR=0
+  PATCH=0
+elif [ "$1" == "minor" ]; then
+  MINOR=$((MINOR+1))
+  PATCH=0
+elif [ "$1" == "patch" ]; then
+  PATCH=$((PATCH+1))
+else
+  echo "Invalid version type. Use 'major', 'minor' or 'patch'"
+  exit 1
+fi
+
+NEW_VERSION="$MAJOR.$MINOR.$PATCH"
+
+# Update extension version
+sed -i -E "s/(\"version\"):.*/\1: \"$NEW_VERSION\",/" vss-extension.json
```

Comment:
Why updating this? Isn't it irrelevant?

Reply:
This script will be used by the Release Automation workflows, and it will always bump both the extension and the task(s) at the same time, staying in sync forever.

The version of the extension doesn't matter. This is just a "container", and users cannot pin its version. The only version that users can pin is the one of the task.

---

Now, if your question is "why are we starting to bump the version in this `vss-extension.json` file also, although we never did it?", then the answer is:

This is to keep a trace of the extension version in the code too. Not only in tags or in the marketplace. The source of truth will now be the tag (to follow our Release Automation conventions), but at least we don't have a useless and misleading `"version": "0.0.1"` field anymore.